dev-libs/libusb-compat/libusb-compat-0.1.5-r2.ebuild

```diff
@@ -36,13 +33,13 @@ multilib_src_configure() {
 }
 
 multilib_src_install() {
-	emake DESTDIR="${D}" install
+	emake DESTDIR="${ED}" install
```

Comment:
This is wrong because `emake` already handles `${EPREFIX}`. Please remove this chunk.